app/views/documents/fields/_govspeak_input.html.erb

```diff
@@ -11,7 +11,7 @@
         id: "govspeak-editor",
         name: "document[contents][#{schema.id}]",
         value: document.contents[schema.id],
-        rows: 30,
+        rows: 25,
```

Comment:
Why has this changed?

Reply:
There isn't a fixed value for this. I initially set this arbitrary at 30, but it seems to be to tall. Will discuss with the designer to figure out what will be a 'good' height for the editor.